settings.py

```diff
@@ -15,7 +15,1348 @@
     # when the code runs on the admin node during deploy
     IS_STAGE = CHIEF_REMOTE_HOSTNAME == 'developer.allizom.org'
 
+<<<<<<< 79a9d5637ab4fca2d785d624516dabcf950d6051
 if IS_STAGE:
     from kuma.settings.stage import *  # noqa
 else:
     from kuma.settings.prod import *  # noqa
+=======
```

Comment:
So screwy. I did a rebase, hit a merge conflict, fixed it just fine and its as if the rebase didn't finish or something weird.